citi-onboarding/maracatronics
# Feature/banner and footer sections

## What i did
- Banner section creation;
- Footer section creation;
- Both section connection with Keystone;

## How to test
1. First step you have to use `git fetch --all` command on terminal;
2. Second change your branch to feature/banner-and-footer-section with `git checkout feature/banner-and-footer-section;`
3. After, open two terminals, one for the server and another for client and use `cd server` and `cd client` commands;
4. Then you have to use `nodemon index.js` on server terminal and `yarn/npm start` on client terminal;
5. In your browser open two pages: `http://localhost:3001/admin` and `http://localhost:3000/` ;
6. On Keystone page try to change Banner part;
7. Reload `http://localhost:3000/` and check if the banner was modified;
8. Also, check the site responsiveness. It's expected that with the screen decrease, the banner remains centralized and the Footer margin change;
